darwin/cli_functions.py

```diff
@@ -966,7 +988,7 @@ def convert(format: str, files: List[PathLike], output_dir: Path) -> None:
     except AttributeError:
         _error(f"Unsupported export format, currently supported: {export_formats}")
 
-    export_annotations(parser, files, output_dir)
+    export_annotations(parser, files, output_dir, split_sequences=(format != "darwin_1.0"))
```

Comment:
If we're changing the format name I would probably favour a different naming convention such as `darwin_v1`, The use of underscores and full stops interchangeably in these string names is likely to cause issues.

Reply:
Also, should this change affect the supported formats list? Does this also have effects on busybee code used for conversion etc?

Reply:
We never had conversion into darwin, as it doesn't make any sense. This a new format, so no, it doesn't affect busybee. Also, I don't want this on a list of supported formats tbh, I don't think that it's a good idea to have legacy stuff like that so openly available. This is a fallback that is supposed to be used:
* by us in busybee to support legacy format in V2 dataset **temporarily** 
* by customers that create exports via CLI if they really, really need legacy format, also **temporarily**

Reply:
As for name - please post this on the channel on Slack `tmp-darwin-json-2`. The name that we have here was discussed in quite wide audience, I don't feel like overriding this decision without discussing with others that initially approved this. 
I don't really buy argument about full stops being an issue, I've never seen that yet myself. And full stop here isn't used interchangeably with underscore - those mean different things. Thus, let's please discuss that with wider audience.